src/wwwroot/app/main.js

```diff
@@ -0,0 +1,16 @@
+import angular from 'angular';
+import router from 'angular-ui-router';
+
+import config from './botConfig';
+import HttpService from './services/httpService';
+import CandidateService from './services/candidateService';
+import VacancyService from './services/vacancyService';
+
+var dependencies = [ router ];
+
+angular
+   .module('bot', dependencies)
+   .service('HttpService', HttpService)

```

Comment:
please, use one empty line to separate different logic blocks (between services, configs, directives, etc)